### DISCUSSION TITLE
Dark title bar

### DISCUSSION THREAD

**Discussion by anoduck** [neovide/neovide#1662] [Q&A] [answered]
I have seen Neovide with a dark title bar before, but on my installation it is solid white. Since I am using a laptop in a low light environment, this white bar beams out and makes using Neovide irritating. Is there anyway this can be changed?

**Comment by MultisampledNight** [comment, marked as answer]
Hi! This is unfortunately a side-effect of Neovide being stuck on an old winit version which renders these suboptimal decorations, you can read more about it at #728 and the first bullet point in #1463. It's unfortunately unfixable for now.

**Reply by anoduck** [reply]
Don't get me wrong, Neovide is totally fricking unbelievably awesome. It is definitely something worthy of writing home about. It is just, the white is almost blinding.

This makes sense, I am running neovide from Google ChromeOS linux subsystem.
